sites/cheerpx/src/content/docs/20-faq.md

```diff
@@ -54,3 +54,19 @@ Cross-Origin-Opener-Policy: same-origin
 ```
 
 By adding these headers to your server configuration you will enable cross-origin isolation and CheerpX will be able to start.
+
+## Why Can't I Use the `file://` Protocol?
+
+CheerpX requires certain browser capabilities that are not available when accessing files directly using the `file://` protocol. Here are the key reasons:
+
+### 1. Security Restrictions
+
+Modern browsers enforce strict security policies that limit how local files can interact with web technologies. Accessing files through `file://` doesn’t allow the necessary cross-origin headers to be enabled, which are crucial for CheerpX to function properly.
+
+### 2. Cross-Origin Headers
```

Comment:
Consider collapsing these 2 points into a single paragraph. I am not convinced either way, so choose what seems to work better